packages/Textarea/tests/Textarea.spec.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render } from "react-testing-library";
+import Textarea from "../src/Textarea";
+
+describe("Textarea", () => {
+  it("should be visible", () => {
+    const { getByText } = render(<Textarea value="placeholder value" />);
+    expect(getByText(/placeholder value/i)).toBeInTheDocument();
+  });
```

Comment:
Is it possible to test the behaviour of the `canExpand` prop? Although, maybe it would be easier/better to add that to the Screener story test?

Reply:
I'll include in screener test, its kinda hard to test here